packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -246,6 +263,50 @@ export const prepareSettler = (
           assert.typeof(registration, 'object');
           this.state.registration = registration;
         },
+        migrateMintedEarly() {
+          const { self } = this.facets;
+          const { mintedEarly } = this.state;
+          for (const [key, count] of mintedEarly.entries()) {
+            const { address, amount } = parseMintedEarlyKey(key);
+            for (let i = 0; i < count; i += 1) {
+              self.addMintedEarly(address, AmountMath.make(USDC, amount));
+            }
+          }
+          mintedEarly.clear();
+        },
+        remediateMintedEarly() {
+          const { self } = this.facets;
+          const { mintedEarly } = this.state;
+          console.log('remediateMintedEarly', [...mintedEarly.keys()]);
+          const batches = mintedEarly.entries();
+          for (const [key, count] of batches) {
+            const { address, amount } = parseMintedEarlyKey(key);
```

Comment:
Thoughts on an early `continue` if `amount < 10 * DENOM_UNIT` ? I think this safely includes things we're actually interested in while excluding test transactions from beta (typically < $1, but suggesting $10 for safety)

Reply:
What's the benefit? Imo production code shouldn't make such assumptions about what is a test value